src/fcntl.rs

```diff
@@ -1,109 +1,24 @@
-
-        #[repr(C)]
-        #[derive(Clone, Copy, Default, Debug)]
-        pub struct flock {
-            pub l_type: c_short,
-            pub l_whence: c_short,
-            pub l_start: off_t,
-            pub l_len: off_t,
-            pub l_pid: pid_t,
-
-            // not actually here, but brings in line with freebsd
-            pub l_sysid: c_int,
-        }
-
-        pub const F_DUPFD:         c_int = 0;
-        pub const F_DUPFD_CLOEXEC: c_int = 1030;
-        pub const F_GETFD:         c_int = 1;
-        pub const F_SETFD:         c_int = 2;
-        pub const F_GETFL:         c_int = 3;
-        pub const F_SETFL:         c_int = 4;
-        pub const F_SETLK:         c_int = 6;
-        pub const F_SETLKW:        c_int = 7;
-        pub const F_GETLK:         c_int = 5;
-
-        pub const F_ADD_SEALS:     c_int = 1033;
-        pub const F_GET_SEALS:     c_int = 1034;
-
-        pub const F_SEAL_SEAL:     c_int = 1;
-        pub const F_SEAL_SHRINK:   c_int = 2;
-        pub const F_SEAL_GROW:     c_int = 4;
-        pub const F_SEAL_WRITE:    c_int = 8;
-    }
-
-    #[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "dragonfly", target_os = "ios", target_os = "openbsd", target_os = "netbsd"))]
-    mod os {
-        use libc::{c_int, c_short, off_t, pid_t};
-
-        #[repr(C)]
-        #[derive(Clone, Copy, Default, Debug)]
-        pub struct flock {
-            pub l_start: off_t,
-            pub l_len: off_t,
-            pub l_pid: pid_t,
-            pub l_type: c_short,
-            pub l_whence: c_short,
-
-            // not actually here, but brings in line with freebsd
-            pub l_sysid: c_int,
-        }
+    use libc::c_int;
 
-        pub const F_DUPFD:         c_int = 0;
-        #[cfg(not(any(target_os = "dragonfly", target_os = "netbsd")))]
-        pub const F_DUPFD_CLOEXEC: c_int = 67;
-        #[cfg(target_os = "dragonfly")]
-        pub const F_DUPFD_CLOEXEC: c_int = 17;
-        #[cfg(target_os = "netbsd")]
-        pub const F_DUPFD_CLOEXEC: c_int = 12;
-        pub const F_GETFD:         c_int = 1;
-        pub const F_SETFD:         c_int = 2;
-        pub const F_GETFL:         c_int = 3;
-        pub const F_SETFL:         c_int = 4;
-        #[cfg(target_os = "netbsd")]
-        pub const F_GETOWN:        c_int = 5;
-        #[cfg(target_os = "netbsd")]
-        pub const F_SETOWN:        c_int = 6;
-        pub const F_GETLK:         c_int = 7;
-        pub const F_SETLK:         c_int = 8;
-        pub const F_SETLKW:        c_int = 9;
-
-        #[cfg(target_os = "netbsd")]
-        pub const F_CLOSEM:        c_int = 10;
-        #[cfg(target_os = "netbsd")]
-        pub const F_MAXFD:         c_int = 11;
-        #[cfg(target_os = "netbsd")]
-        pub const F_GETNOSIGPIPE:  c_int = 13;
-        #[cfg(target_os = "netbsd")]
-        pub const F_SETNOSIGPIPE:  c_int = 14;
-    }
+    pub const F_ADD_SEALS: c_int = 1033;
```

Comment:
add a TODO explaining why these are left here, and reference https://github.com/rust-lang/libc/issues/235